test/fx/test_gradual_type.py

```diff
@@ -819,6 +823,53 @@ def test_resnet50(self):
         for n1, n2 in zip(gm_static_with_types.graph.nodes, gm_run.graph.nodes):
             assert n1.type == TensorType(n2.meta['tensor_meta'].shape)
 
+        # apply shape inference to graph and check
+        # that the batch size is equal across all layers
+        infer_symbolic_types(gm_static)
+        infer_symbolic_types(gm_static)
```

Comment:
you mentioned elsewhere that you need to run it twice - should the be invoked internally as part of `infer_symbolic_types` instead of being required at the callsite ?

Reply:
yeah I will do that. The tests are calling infer_symbolic_types twice. I'll change that to a single call.